src/zelos/config_gen.py

```diff
@@ -138,8 +141,65 @@ def generate_parser():
         "--traceon",
         type=str,
         default="",
-        help="[Experimental] Enable verbose tracing after specified address "
-        "or API name.",
+        help=(
+            "[Experimental] Enable verbose tracing after specified"
+            "address or API name."
+        ),
+    )
+    group_feeds.add_argument(
+        "--inst_feed",
+        action="append",
+        nargs="?",
+        default=[],
+        const="",
+        help=(
+            "If used, turns on the inst feed. A string containing "
+            "conditions for the inst feed to turn on can be specified."
+            "Multiple triggers can be specified by using this flag"
+            "multiple times."
+        ),
+    )
+
+    group_feeds.add_argument(
+        "--api_feed",
+        action="append",
+        nargs="?",
+        default=[],
+        const="",
+        help=(
+            "If used, turns on the api feed. A string containing "
```

Comment:
Not clear that this enables other lower-level feeds. Clarify something like this:

"If used, sets the feed level to API. A ZML condition string can be specified as a trigger to set the level. Multiple triggers..."

Feeds/Levels should be explained in a tutorial as well.

Reply:
Reworded